src/services/provider/oaf.provider.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @module services/provider/oaf_provider
+ */
+/**
+ * BVV specific implementation of {@link module:services/ImportOafService~oafGeoResourceProvider}
+ * @function
+ * @type {module:services/ImportOafService~oafFilterCapabilitiesProvider}
+ */
+
+import { OafGeoResource } from '../../domain/geoResources';
+
+// eslint-disable-next-line no-unused-vars
+export const bvvOafFilterCapabilitiesProvider = async (oafGeoResource) => {
```

Comment:
I guess we need a hint here as comment (//HINT:), that this provider is currently static and will change when the filter capabilities are or can be properly utilised.